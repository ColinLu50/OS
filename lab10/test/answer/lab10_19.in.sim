24998
1160
1524
894
1522
887
