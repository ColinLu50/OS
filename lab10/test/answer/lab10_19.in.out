FCFS
629 -> 394 -> 532 -> 759 -> 721 -> 436 -> 349 -> 466 -> 826 -> 12 -> 711 -> 562 -> 191 -> 1 -> 109 -> 428 -> 895 -> 561 -> 775 -> 506 -> 449 -> 486 -> 245 -> 391 -> 472 -> 437 -> 68 -> 466 -> 734 -> 219 -> 118 -> 465 -> 88 -> 539 -> 715 -> 371 -> 235 -> 181 -> 823 -> 863 -> 782 -> 328 -> 675 -> 636 -> 191 -> 432 -> 20 -> 799 -> 555 -> 662 -> 817 -> 513 -> 706 -> 523 -> 129 -> 94 -> 118 -> 855 -> 396 -> 140 -> 342 -> 417 -> 266 -> 177 -> 713 -> 719 -> 497 -> 386 -> 800 -> 58 -> 125 -> 364 -> 352 -> 465 -> 182 -> 77 -> 839 -> 487 -> 829 -> 771 -> 25 -> 578 -> 195 -> 880 -> 853 -> 293 -> 125 -> 12 -> 101 -> 34 -> 294 -> 298 -> 307 -> 36 -> 93
24998
SSTF
629 -> 636 -> 662 -> 675 -> 706 -> 711 -> 713 -> 715 -> 719 -> 721 -> 734 -> 759 -> 771 -> 775 -> 782 -> 799 -> 800 -> 817 -> 823 -> 826 -> 829 -> 839 -> 853 -> 855 -> 863 -> 880 -> 895 -> 578 -> 562 -> 561 -> 555 -> 539 -> 532 -> 523 -> 513 -> 506 -> 497 -> 487 -> 486 -> 472 -> 466 -> 466 -> 465 -> 465 -> 449 -> 437 -> 436 -> 432 -> 428 -> 417 -> 396 -> 394 -> 391 -> 386 -> 371 -> 364 -> 352 -> 349 -> 342 -> 328 -> 307 -> 298 -> 294 -> 293 -> 266 -> 245 -> 235 -> 219 -> 195 -> 191 -> 191 -> 182 -> 181 -> 177 -> 140 -> 129 -> 125 -> 125 -> 118 -> 118 -> 109 -> 101 -> 94 -> 93 -> 88 -> 77 -> 68 -> 58 -> 36 -> 34 -> 25 -> 20 -> 12 -> 12 -> 1
1160
SCAN
629 -> 578 -> 562 -> 561 -> 555 -> 539 -> 532 -> 523 -> 513 -> 506 -> 497 -> 487 -> 486 -> 472 -> 466 -> 465 -> 449 -> 437 -> 436 -> 432 -> 428 -> 417 -> 396 -> 394 -> 391 -> 386 -> 371 -> 364 -> 352 -> 349 -> 342 -> 328 -> 307 -> 298 -> 294 -> 293 -> 266 -> 245 -> 235 -> 219 -> 195 -> 191 -> 182 -> 181 -> 177 -> 140 -> 129 -> 125 -> 118 -> 109 -> 101 -> 94 -> 93 -> 88 -> 77 -> 68 -> 58 -> 36 -> 34 -> 25 -> 20 -> 12 -> 1 -> 0 -> 636 -> 662 -> 675 -> 706 -> 711 -> 713 -> 715 -> 719 -> 721 -> 734 -> 759 -> 771 -> 775 -> 782 -> 799 -> 800 -> 817 -> 823 -> 826 -> 829 -> 839 -> 853 -> 855 -> 863 -> 880 -> 895
1524
C-SCAN
629 -> 578 -> 562 -> 561 -> 555 -> 539 -> 532 -> 523 -> 513 -> 506 -> 497 -> 487 -> 486 -> 472 -> 466 -> 465 -> 449 -> 437 -> 436 -> 432 -> 428 -> 417 -> 396 -> 394 -> 391 -> 386 -> 371 -> 364 -> 352 -> 349 -> 342 -> 328 -> 307 -> 298 -> 294 -> 293 -> 266 -> 245 -> 235 -> 219 -> 195 -> 191 -> 182 -> 181 -> 177 -> 140 -> 129 -> 125 -> 118 -> 109 -> 101 -> 94 -> 93 -> 88 -> 77 -> 68 -> 58 -> 36 -> 34 -> 25 -> 20 -> 12 -> 1 -> 0 -> 901 -> 895 -> 880 -> 863 -> 855 -> 853 -> 839 -> 829 -> 826 -> 823 -> 817 -> 800 -> 799 -> 782 -> 775 -> 771 -> 759 -> 734 -> 721 -> 719 -> 715 -> 713 -> 711 -> 706 -> 675 -> 662 -> 636
894
LOOK
629 -> 578 -> 562 -> 561 -> 555 -> 539 -> 532 -> 523 -> 513 -> 506 -> 497 -> 487 -> 486 -> 472 -> 466 -> 465 -> 449 -> 437 -> 436 -> 432 -> 428 -> 417 -> 396 -> 394 -> 391 -> 386 -> 371 -> 364 -> 352 -> 349 -> 342 -> 328 -> 307 -> 298 -> 294 -> 293 -> 266 -> 245 -> 235 -> 219 -> 195 -> 191 -> 182 -> 181 -> 177 -> 140 -> 129 -> 125 -> 118 -> 109 -> 101 -> 94 -> 93 -> 88 -> 77 -> 68 -> 58 -> 36 -> 34 -> 25 -> 20 -> 12 -> 1 -> 636 -> 662 -> 675 -> 706 -> 711 -> 713 -> 715 -> 719 -> 721 -> 734 -> 759 -> 771 -> 775 -> 782 -> 799 -> 800 -> 817 -> 823 -> 826 -> 829 -> 839 -> 853 -> 855 -> 863 -> 880 -> 895
1522
C-LOOK
629 -> 578 -> 562 -> 561 -> 555 -> 539 -> 532 -> 523 -> 513 -> 506 -> 497 -> 487 -> 486 -> 472 -> 466 -> 465 -> 449 -> 437 -> 436 -> 432 -> 428 -> 417 -> 396 -> 394 -> 391 -> 386 -> 371 -> 364 -> 352 -> 349 -> 342 -> 328 -> 307 -> 298 -> 294 -> 293 -> 266 -> 245 -> 235 -> 219 -> 195 -> 191 -> 182 -> 181 -> 177 -> 140 -> 129 -> 125 -> 118 -> 109 -> 101 -> 94 -> 93 -> 88 -> 77 -> 68 -> 58 -> 36 -> 34 -> 25 -> 20 -> 12 -> 1 -> 895 -> 880 -> 863 -> 855 -> 853 -> 839 -> 829 -> 826 -> 823 -> 817 -> 800 -> 799 -> 782 -> 775 -> 771 -> 759 -> 734 -> 721 -> 719 -> 715 -> 713 -> 711 -> 706 -> 675 -> 662 -> 636
887
